src/common/webrtc/api.rs

```diff
@@ -426,6 +431,17 @@ where
             .get(0)
             .ok_or_else(|| WebRtcError::InvalidSDPOffer("no media description".to_owned()))?;
 
+        let remote_prio = attribute
```

Comment:
[q] what determines the priority number from the other side of the connection? what logic is meant to be used to determine it?

Reply:
there is no logic for now, only the RC page will sent a priority value. I don't have an idea as too how we are going to deal with it moving forward